macOS/DuckDuckGo/NetworkProtection/AppTargets/DeveloperIDTarget/NetworkProtectionIPCTunnelController.swift

```diff
@@ -108,11 +108,14 @@ extension NetworkProtectionIPCTunnelController: TunnelController {
 
             knownFailureStore.reset()
 
-            ipcClient.start { [pixelKit] error in
-                if let error {
-                    handleFailure(error)
-                } else {
-                    pixelKit?.fire(StartAttempt.success, frequency: .legacyDailyAndCount)
+            try await withCheckedThrowingContinuation { (continuation: CheckedContinuation<Void, Error>) in
+                ipcClient.start { [pixelKit] error in
+                    if let error {
+                        continuation.resume(throwing: error)
+                    } else {
+                        pixelKit?.fire(StartAttempt.success, frequency: .legacyDailyAndCount)
+                        continuation.resume()
+                    }
```

Comment:
This change is because previously we would make an IPC request to the menu app to start the VPN, and return immediately.  Now I'm changing the approach so that our code will actually await for the IPC call to return, which means we'll wait until the menu app's VPN start attempt succeeds or fail.

The reason this change was necessary was to hold the toggle in the ON position while the VPN tunnel is still disconnected.